Resolve issue#23: Explain the sources of llbmc-bench and GNU Coreutils

feliciahalim opened the pull request:
To resolve issue in here: https://github.com/feliciahalim/klee-examples/issues/23

feliciahalim committed:
Resolve issue#23: Explain the sources of llbmc-bench and GNU Coreutils

domainexpert commented:
@feliciahalim Awesome! I would suggest removing the phrase "which form the core user-level environment installed on millions of Unix systems", as this is widely known.

feliciahalim committed:
Remove explanation in Coreutils-6.10

Remove "which form the core user-level environment installed on millions of Unix systems" as it is widely known.

feliciahalim commented:
@domainexpert I have removed the phrase "which form the core user-level environment installed on millions of Unix systems" in Coreutils-6.10 explanation.

domainexpert commented:
@feliciahalim Looks good to me. Please merge if you wish.
